tests/functional/snowflake_view_dependency/test_snowflake_view_dependency.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from dbt.tests.util import run_dbt, check_relations_equal
+
+_MODELS__DEPENDENT_MODEL_SQL = """
+
+{% if var('dependent_type', 'view') == 'view' %}
+    {{ config(materialized='view') }}
+{% else %}
+    {{ config(materialized='table') }}
+{% endif %}
+
+select * from {{ ref('base_table') }}
+
+"""
+
+
+
+_MODELS__BASE_TABLE_SQL = """
+
+{{ config(materialized='table') }}
+
+select *
+    {% if var('add_table_field', False) %}
+        , 1 as new_field
+    {% endif %}
+
+from {{ ref('people') }}
+
+"""
+
+_SEEDS__PEOPLE_CSV = """id,name
+1,Drew
+2,Jake
+3,Connor
+"""
+
+class TestSnowflakeLateBindingViewDependency:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "dependent_model.sql":  _MODELS__DEPENDENT_MODEL_SQL,
+            "base_table.sql": _MODELS__BASE_TABLE_SQL
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"people.csv": _SEEDS__PEOPLE_CSV}
+
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            'config-version': 2,
```

Comment:
```suggestion
```

The line is redundant. I think `seed-paths` is too since these are in the default fixture (go check them out to be sure)